samples/react-native/e2e/captureTransaction.test.ts

```diff
@@ -0,0 +1,190 @@
+import { describe, it, beforeAll, expect, afterAll } from '@jest/globals';
+import { EventItem } from '@sentry/core';
+import {
+  createSentryServer,
+  containingTransactionWithName,
+} from './utils/mockedSentryServer';
+
+import { getItemOfTypeFrom } from './utils/event';
+import { maestro } from './utils/maestro';
+import { isAndroid, isIOS } from './utils/environment';
```

Comment:
Removing this import should please the linter

Reply:
Thank you. I missed that.